.buildkite/pipeline.yml

```diff
@@ -0,0 +1,100 @@
+agents:
+  queue: k8s-builders
+
+steps:
+  - label: lint
+    key: lint
+    command: make lint
+    plugins:
+      - docker#v5.11.0:
+          image: "golang:1.22"
+          environment:
+            - BUILDKITE=true
+  - label: unit
+    key: unit
+    command: make unit
+    plugins:
+      - docker#v5.11.0:
+          image: "golang:1.22"
+    depends_on:
+      - lint
```

Comment:
I think I already asked, sorry that I forgot. Will these run no matter what? or does linting + testing need the PR to be tagged as well?

Reply:
There's a relatively recently added buildkite feature that makes it easy to detect labels on PRs so I've taken advantage of that in the latest commit. When the label is missing lint and unit will run. When it is added lint->unit->network->rest of integration will run.